environment.yml

```diff
@@ -105,3 +104,7 @@ dependencies:
   - factory_boy
   - flake8
   - flake8-bugbear
+
+  # reactpy dependencies
+  - pip:
+    - reactpy-django
```

Comment:
I've been looking into that. Unfortunately, it looks like there may be 4 total packages we'd need to add, since all of the dependencies also need to be in conda-forge. Those 4 packages are: reactpy-django, reactpy, reactpy-router, and nodejs-bin...

Reply:
That's pretty typical for adding conda packages. I think you might be able to add them all in one go if they are related...

Reply:
I went ahead and added `reactpy` to the dependencies